hypergan/loaders/image_loader.py

```diff
@@ -14,16 +14,21 @@ def build_labels(dirs):
     labels[dir.split('/')[-1]]=next_id
     next_id+=1
   return labels,next_id
-def labelled_image_tensors_from_directory(directory, batch_size, channels=3, format='jpg', width=64, height=64, crop=False, preprocess=False):
-  filenames = glob.glob(directory+"/**/*."+format)
-  labels,total_labels = build_labels(sorted(filter(os.path.isdir, glob.glob(directory+"/*"))))
+def labelled_image_tensors_from_directory(directory, batch_size, channels=3, format='jpg', width=64, height=64, crop=False, preprocess=False, filterX=None):
```

Comment:
If we don't pass in a filterX, could we just pass in different directories?

```bash
So it would be structured like:

imagedir1/a/
imagedir2/b/
```
---
Also I wonder if we should separate image alignment into another PR (or set of PRs).   There are many ways to align that need to be explored. (discoGAN, cycleGAN)